build/tsconfig.commonjs.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.common.json",
   "compilerOptions": {
+    "isolatedModules": false,
```

Comment:
this config and the next one are used for `yarn build` scenarios, where Babel is not used